drivers/hmis/app/graphql/types/forms/pick_list_option.rb

```diff
@@ -53,6 +53,11 @@ def self.options_for_type(pick_list_type, user:, project_id: nil, client_id: nil
           preload(:organization).
           sort_by_option(:organization_and_name).
           map(&:to_pick_list_option)
+      when 'OPEN_PROJECTS'
+        Hmis::Hud::Project.viewable_by(user).where(operating_end_date: nil).
```

Comment:
A project can still be open if the operating end date is in the future. We can use `open_on_date` scope to get open projects

Reply:
Sounds good -- this is what I wrote originally but I noticed when I set a project's end date to Date.current, it was still refer-able. Maybe want `open_on_date(Date.current + 1)`?

As an aside, is there a way to close a project from the UI? I was just doing it in the db

Reply:
Looks like we use `gteq` in that scope, the idea being that the project is open during some part of the day, maybe, but who knows. I don't think it hurts to include those projects in the list, it should be infrequent. I think lets just use `open_on_date`. In theory if its the last day of the project, someone could still enroll and ext that same day if they receive one-day service.

Yep you can close it in the UI by editing the operating end date in the project details form.